api/v1_queues.inc

```diff
@@ -0,0 +1,161 @@
+<?php
+include_once($relPath.'user_is.inc');
+include_once("exceptions.inc");
+
+// TODO(bpfoley): There is a lot of duplicated logic betweeen this file and
+//       stats/release_queue.php's _show_round_queues
+// TODO(bpfoley): Add openAPI schema
+// NB because of how the queue_defns table is read in order, and queues
```

Comment:
<nod> At this point I wouldn't worry about pagination for the (good) reasons you've highlighted.